docs/source/architecture.rst

```diff
@@ -0,0 +1,143 @@
+Drivers Atlas Testing Architecture
+==================================
+
+The Drivers Atlas Testing project validates the behavior of drivers against
+Atlas clusters. For example, a test could be to perform queries or writes
+in a loop while Atlas performs a maintenance operation, and assert that all
+queries and writes complete successfully.
+
+The project consists of the following principal components:
+
+- Astrolabe - the orchestration utility.
+- :doc:`Workload executors <spec-workload-executor>` - wrappers providing
+  a common interface to each driver's unified test runner.
+- Each driver's unified test runner.
+- :doc:`The test scenario specification <spec-test-format>`.
+
+Astrolabe
+---------
+
+Users invoke ``astrolabe`` to perform a test. Astrolabe interacts with Atlas
+and it performs several common tasks to keep the workload executors as small
+and simple as possible. Specifically, Astrolabe:
+
+- Creates an Atlas cluster for testing.
+- Launches a workflow executor pointed at the cluster.
```

Comment:
Launches a workflow executor pointed at the cluster once the cluster has been provisioned and is ready for operations.*